docs/developers/nft-tokens.md

```diff
@@ -261,6 +263,76 @@ TransferCreationRoleTransaction {
 }
 ```
 
+### **Stop NFT creation**
+
+The ESDT manager can stop the creation of an NFT for the given ESDT forever, by removing the only `ESDTRoleNFTCreate` role available.
+This is done by performing a transaction like this:
+
+```
+StopNFTCreationTransaction {
+    Sender: <account address of the token manager>
+    Receiver: erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u
+    Value: 0
+    GasLimit: 500000 + length of Data field in bytes * 1500
+    Data: "stopNFTCreate" +
+          "@" + <token identifier in hexadecimal encoding> +
+}
+```
+
+### **Freezing and Unfreezing a single NFT**
```

Comment:
`###` already makes the text bold, so there is no need for `**`. Please remove `**` from all section / subsection titles when you can.

Reply:
true, but it will be displayed with no bold in the sidebar menu. just kept the consistency